disco_aws_automation/disco_remote_exec.py

```diff
@@ -197,7 +197,17 @@ def rsync(address, source, destination, user, nothrow=False):
     @staticmethod
     def _call_rsync(command):
         if logging.getLogger().getEffectiveLevel() == logging.DEBUG:
-            return subprocess.call(command)
+            with tempfile.TemporaryFile() as output:
+                process = subprocess.Popen(
+                    command,
+                    stdout=output,
+                    stderr=subprocess.STDOUT
+                )
+                process.communicate()
+                output.seek(0)
+                stdout = output.read()
+                logger.debug(stdout)
+                return process.returncode
```

Comment:
Hmm. Is this actually logging when it's debug now then?

Reply:
`if logging.getLogger().getEffectiveLevel() == logging.DEBUG`

Before my change it was spitting out the command output in stdout.

Reply:
I see.